orchestrator/package.json

```diff
@@ -1,18 +1,41 @@
 {
   "name": "orchestrator",
-  "version": "1.0.0",
+  "version": "1.1.0",
   "description": "Flow orchestrator module",
   "main": "src/index.js",
   "scripts": {
-    "test": ""
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "check-licenses": "apache2-license-checker"
```

Comment:
>     "check-licenses": "apache2-license-checker"

Don't you need to include this in development dependencies?

Reply:
You're totally right!